helm/templates/deployment.yaml

```diff
@@ -71,6 +76,10 @@ spec:
           image: {{ $cloudProviderDockerRegistryUrl }}{{ .repository }}:{{ .tag }}
           {{- end }}
           env:
+            {{- if .Values.rasterCommon.ca.secretName }}
+            - name: SSL_CERT_DIR
+              value: /usr/local/share/ca-certificates/
```

Comment:
Make this configureable like the mount below.

Reply:
fixed

Reply:
fixed